### DISCUSSION TITLE
Why did you remove `--instruct` from the parameters?

### DISCUSSION THREAD

**Discussion by gjnave** [ggml-org/llama.cpp#7998] [Q&A]
Just curious why thats not available any longer... always brought stability to my models. What's the equivalent now for chat-back-and-forth mode?
Thanks

**Comment by sysadminjohn** [comment]
I was wondering the same thing. I am using `-i -cnv` instead of `-i -ins`, but something feels a bit off from the usual behaviour. Can't say if for better or worse, I guess they just do different things.

Let's say instruct mode tried to execute my requests and outputted a long bunch of text, before giving me back control. Now it tends to give me back control quite more often, sometimes annoyingly often, to the point of completely missing the point of my request, but only in certain situations.

It's as if it attempts to continue the prompt, rather than execute its instructions

**Comment by ericcurtin** [comment]
Also opened an issue here:

https://github.com/ggerganov/llama.cpp/issues/8681
